backend/graphql/schema.graphql

```diff
@@ -106,6 +153,42 @@ type User {
     email_verified_at: DateTime
     roles: [Role!]! @belongsToMany
     permissions: [Permission!] @belongsToMany
+    profile_metadata: ProfileMetadata
+}
+
```

Comment:
Individual subtypes should have a docstring on them.  Every field would be overkill, but at least the subtypes.

Reply:
Added.